plugins/woocommerce/src/RestApi/Routes/V4/OfflinePaymentMethods/OfflinePaymentMethodSchema.php

```
@@ -0,0 +1,249 @@
+		);
+	}
+
+	/**
+	 * Get the item response.
+	 *
+	 * @param mixed           $item Payment method data array.
+	 * @param WP_REST_Request $request Request object.
+	 * @param array           $include_fields Fields to include in the response.
+	 * @return array The item response.
+	 */
+	public function get_item_response( $item, WP_REST_Request $request, array $include_fields = array() ): array {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)